### DISCUSSION TITLE
Spacy: interaction with disks once the model loaded

### DISCUSSION THREAD

**Discussion by bissana** [explosion/spaCy#8135] [Help: Other Questions] [answered]
Hi,
I'm a new user. I am happily using spacy on my personal laptop which has SSD storage. The same application is 3 times slower on another machine that has the same system (Ubuntu), CPUs and RAM but SATA instead of SSD. 
Once the model is loaded does spacy interact with the disks to tokenise, parse, ... or all these operations should perform the same whether the system has SSD or not?
Thank you

**Comment by adrianeboyd** [comment, marked as answer]
The models are loaded in memory, so the speed for running a text through a pipeline shouldn't depend on disk access.

If everything else is identical, double-check that you have the exact same versions of all the dependencies, in particular `blis`, where you may notice a big difference in speed on newer CPUs (v0.4.x vs. v0.7.x).

**Reply by bissana** [reply]
Thank you !
